downstream/attributes/attributes.adoc

```diff
@@ -28,8 +28,9 @@
 
 
 // Automation services catalog
-:CatalogName: automation services catalog
-:CatalogNameStart: Automation services catalog
+// Hherbly--removed attributes for catalog since it is deprecated for AAP 2.4. See: AAP-12573
```

Comment:
@hherbly 
Just my own opinion, not backed up by anything, but I think it might be a good idea to keep the Catalog attributes.
That way, we can use them in docs to mention that Catalog has been deprecated.

Reply:
That makes sense--thanks @ariordan-redhat. I'll change it back.